### ISSUE TITLE
The Clock applet doesn't match menubar buttons

### ISSUE THREAD

**Issue by flexiondotorg** [ubuntu-mate/ubuntu-mate-artwork#2]
The Clock applet button is not styled the same as Applications Places System. There should be no visual change when hovering over the Clock applet and when click it the button style should match Application Place System. This issue exists in all 3 themes.

Tagging @raveit65

**Comment by raveit65** [comment]
Well, the menubar (Applications Places System) use general settings for menus and isn't an applet in my (css) world.
The clock- applet is an applet and follow the the style from other applets like ie. wnck-list and others.
Also boths have different states.
menubar have only 2, **menubar and menubar.hover** (here hover=selected)
clock use four, **button, button:hover, button:checked (selected) and button:checked:hover**

Sure i can make ** menubar.hover and button:checked** similar, as the clock use his own selector (clock-applet-button), but what should i do with the other states (button:hover and button:checked:hover) ?
Do you really want them un-styled?

Edit:
In short, what you see here is the different between **menubar and button** selectors.

**Comment by flexiondotorg** [comment]
So the styling of button:hover and button:checked:hover should, as closely as possible, resemble menubar behaviour. And I think that button:hover should not change style compared to when not hovering.
